Makefile

```diff
@@ -2,26 +2,26 @@ LISP ?= ${shell which sbcl}
 
 ncurses:
 	qlot install
-	qlot exec $(LISP) --load scripts/build-ncurses.lisp
+	$(LISP) --load .qlot/setup.lisp --load scripts/build-ncurses.lisp
 
 sdl2:
 	qlot install
-	qlot exec $(LISP) --load scripts/build-sdl2.lisp
+	$(LISP) --load .qlot/setup.lisp --load scripts/build-sdl2.lisp
 
 test:
 	qlot install
-	qlot exec .qlot/bin/rove lem-tests.asd
+	.qlot/bin/rove lem-tests.asd
 
 doc:
 	qlot install
-	qlot exec $(LISP) --load scripts/generate-documentation-tests.lisp --eval '(progn (lem-documentation-mode/tests::generate-markdown-file "test.md" :test) (quit))'
+	$(LISP) --load .qlot/setup.lisp --load scripts/generate-documentation-tests.lisp --eval '(progn (lem-documentation-mode/tests::generate-markdown-file "test.md" :test) (quit))'
 
 update:
 	git pull
 	qlot install
 
 lint:
-	qlot exec .qlot/bin/sblint src/base/lem-base.asd
-	qlot exec .qlot/bin/sblint lem.asd
-	qlot exec .qlot/bin/sblint lib/lisp-syntax/lem-lisp-syntax.asd
-	qlot exec .qlot/bin/sblint extensions/lisp-mode/lem-lisp-mode.asd
+	.qlot/bin/sblint src/base/lem-base.asd
+	.qlot/bin/sblint lem.asd
+	.qlot/bin/sblint lib/lisp-syntax/lem-lisp-syntax.asd
+	.qlot/bin/sblint extensions/lisp-mode/lem-lisp-mode.asd
```

Comment:
It's not necessary to run scripts under `.qlot/bin` after `qlot exec` because Qlot takes care of them.